src/Services/ConditionService.php

```diff
@@ -119,7 +119,11 @@ public function createResultRecordFromDatabaseResult($row)
      */
     public function getAll($search = array(), $isAndCondition = true, $puuidBind = null)
     {
-        $newSearch = [];
```

Comment:
Why remove this line?  $newSearch still needs to be defined.

Reply:
This is probably why the condition is failing in the test suite.  Everything else looked fine.

Reply:
Oops, somehow this got in. Fixed 👍